pcsclite: polkit policy actions not registered

### Describe the bug
`pcsc-lite` uses two actions `org.debian.pcsc-lite.access_pcsc` and `org.debian.pcsc-lite.access_pcsc` do get necessary authorization via `polkit`. but these actions are not properly registed, leading to error:

```
journalctl --unit=pcscd --follow
Jan 13 23:34:19 jaghut pcscd[79235]: 00607239 auth.c:125:IsClientAuthorized() Error in authorization: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: Action org.debian.pcsc-lite.access_pcsc is not registered
Jan 13 23:34:19 jaghut pcscd[79235]: 00000005 auth.c:143:IsClientAuthorized() Process 79228 (user: 1000) is NOT authorized for action: access_pcsc
Jan 13 23:34:19 jaghut pcscd[79235]: 00000039 winscard_svc.c:355:ContextThread() Rejected unauthorized PC/SC client
Jan 13 23:34:19 jaghut pcscd[79235]: 00141489 auth.c:125:IsClientAuthorized() Error in authorization: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: Action org.debian.pcsc-lite.access_pcsc is not registered
Jan 13 23:34:19 jaghut pcscd[79235]: 00000005 auth.c:143:IsClientAuthorized() Process 79228 (user: 1000) is NOT authorized for action: access_pcsc
Jan 13 23:34:19 jaghut pcscd[79235]: 00000037 winscard_svc.c:355:ContextThread() Rejected unauthorized PC/SC client
Jan 13 23:34:19 jaghut pcscd[79235]: 00002007 auth.c:125:IsClientAuthorized() Error in authorization: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: Action org.debian.pcsc-lite.access_pcsc is not registered
```
(It can be verified that these actions don't exist by using `pkaction`)

This breaks `yubioauth-flutter` and `gpg`.

I tried both pcsclite 1.9.5 and 2.0.1 ending up with the same error.

### Steps To Reproduce
Steps to reproduce the behavior:
1. Set `services.pcscd.enable = true` and `security.polkit.enable = true` in `configuration.nix`, configure your smartcard (yubikey)
2. Try to use yubikey
3. Get error

### Expected behavior
There actions are registered correctly and usable.

### Notify maintainers

@anthonyroussel 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

I'm using flakes (no channels).

```console
[user@system:~]$  nix-shell -I nixpkgs=./src/nixpkgs -p nix-info --run "nix-info -m"
this path will be fetched (0.00 MiB download, 0.00 MiB unpacked):
  /nix/store/2bk3cqpj26fi3fwgshgg3k0fxwz0h55g-nix-info
copying path '/nix/store/2bk3cqpj26fi3fwgshgg3k0fxwz0h55g-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.0, NixOS, 24.05 (Uakari), 24.05.20240113.76a31fd`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `""`
 - nixpkgs: `not found`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

> I do not think I can fully test this due to the amount of rebuilds, so please chime in for reviews and find the one mistake I might have done (which I hope I didn't)

Tried to build but it fails on my machine:
```
error: builder for '/nix/store/wc590z8yj6cs043rycgz7sd5id0727kg-python3.11-dulwich-0.21.7.drv' failed with exit code 1;
       last 10 log lines:
       > dulwich/tests/test_repository.py::RepositoryRootTests::test_fetch_ignores_missing_refs
       >   /build/dulwich-0.21.7/dulwich/refs.py:1280: UserWarning: ref refs/heads/blah points at non-present sha 1234566789123456789123567891234657373833
       >     warnings.warn(
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > FAILED dulwich/tests/compat/test_client.py::DulwichTCPClientTest::test_fetch_pack_no_side_band_64k - AssertionError: Lists differ: [b'07a4f52db4e20c8e63d58a342eb7f5a23b2b5d7[85...
       > FAILED dulwich/tests/compat/test_client.py::DulwichTCPClientTest::test_send_pack_nothing_to_send - AssertionError: {b'HEAD': b'514dc6d3fbfe77361bcaef320c4d21[141 chars]be9'} ...
       > ===== 2 failed, 1468 passed, 33 skipped, 1 xfailed, 251 warnings in 5.63s ======
       > /nix/store/v5irq7wvkr7kih0hhnch5nnv2dcq8c4f-stdenv-linux/setup: line 1557: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/wc590z8yj6cs043rycgz7sd5id0727kg-python3.11-dulwich-0.21.7.drv'.
error: 1 dependencies of derivation '/nix/store/mqwvy609gvix7nhc1f28nlxyx03m48w5-python3.11-breezy-3.3.4.drv' failed to build
error: 1 dependencies of derivation '/nix/store/q655714kpc1pk5g62r2vggmh5ag67wcj-nix-prefetch-bzr.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9hgcfpwsf3nhi7lasbifjhsrpv9bkam0-nix-prefetch-scripts.drv' failed to build
error: 1 dependencies of derivation '/nix/store/5pgl5jvpcvp0kp6d9hss3h57r06ppnd3-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/0gxnj3603ybw3qirn233xa1857nmjs2x-nixos-system-jaghut-24.05.20240124.ecc103f.drv' failed to build
```

--- a/nixos/modules/services/hardware/pcscd.nix
+++ b/nixos/modules/services/hardware/pcscd.nix
@@ -46,8 +46,8 @@ in
   config = mkIf config.services.pcscd.enable {
     environment.etc."reader.conf".source = cfgFile;
 
-    environment.systemPackages = [ package.out ];
-    systemd.packages = [ (getBin package) ];
+    environment.systemPackages = [ package ];
+    systemd.packages = [ package ];
 
     services.pcscd.plugins = [ pkgs.ccid ];
 
@@ -64,7 +64,7 @@ in
       # around it, we force the path to the cfgFile.
       #
       # https://github.com/NixOS/nixpkgs/issues/121088
-      serviceConfig.ExecStart = [ "" "${getBin package}/bin/pcscd -f -x -c ${cfgFile}" ];
+      serviceConfig.ExecStart = [ "" "${package}/bin/pcscd -f -x -c ${cfgFile}" ];
     };
   };
 }

--- a/pkgs/development/tools/global-platform-pro/default.nix
+++ b/pkgs/development/tools/global-platform-pro/default.nix
@@ -47,7 +47,7 @@ mavenJdk8.buildMavenPackage rec {
     cp tool/target/gp.jar "$out/share/java"
     makeWrapper "${jre8_headless}/bin/java" "$out/bin/gp" \
       --add-flags "-jar '$out/share/java/gp.jar'" \
-      --prefix LD_LIBRARY_PATH : "${pcsclite.out}/lib"
+      --prefix LD_LIBRARY_PATH : "${lib.getLib pcsclite}/lib"
   '';
 
   meta = with lib; {

--- a/pkgs/tools/security/pcsclite/default.nix
+++ b/pkgs/tools/security/pcsclite/default.nix
@@ -21,7 +21,7 @@ stdenv.mkDerivation (finalAttrs: {
   inherit pname;
   version = "2.0.1";
 
-  outputs = [ "bin" "out" "dev" "doc" "man" ];
+  outputs = [ "out" "lib" "dev" "doc" "man" ];
 
   src = fetchFromGitLab {
     domain = "salsa.debian.org";
@@ -39,7 +39,7 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.enableFeature polkitSupport "polkit")
   ] ++ lib.optionals stdenv.isLinux [
     "--enable-ipcdir=/run/pcscd"
-    "--with-systemdsystemunitdir=${placeholder "bin"}/lib/systemd/system"
+    "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
   ];
 
   makeFlags = [

